openedx/features/marketplace/forms.py

```diff
@@ -13,4 +14,8 @@ class Meta:
                 'style': 'pointer-events: none;'
             })
         }
+        labels = {
+            'organization': _('Organization Name'),
+            'organization_mission': _('Organization Mission'),
+        }
```

Comment:
Instead of this approach, wouldn't it be better if we update the OrganizationBase model and introduce verbose_name property in organization and organization_mission field. That would be an in-line solution, and will simplify this forms.py file.

Also, we are using Organization in Idea form as well, and we might have to do add the same labels in Idea form as well. To avoid such code duplication, I think it would be better to directly update the Organization model and use the verbose_name property, as it is used for other fields.

Thoughts @qasimgulzar @ahmed-faraz46?

Reply:
Moreover, I think we need to update the label, like from 'Organization Name' to 'Organization Name*' since it is a mandatory field. Similarly with organization mission. Also, we'll have to add '*' at the end of every verbose_name in the mandatory fields on the MarketplaceRequestModel, so that when they are rendered in the front end, the mandatory fields have a '*' at the end of their label tags.

I discussed with Hassan and he said he is populating the labels received from backend as it is.

Reply:
changing in the model will regenerate the migration, also this has to be linked with frontend so it is good to override labels in form class that's why they have provided the functionality.